torch/distributed/tensor/parallel/__init__.py

```diff
@@ -1,23 +1,20 @@
 # Copyright (c) Meta Platforms, Inc. and affiliates
+from torch.distributed.tensor.parallel.api import parallelize_module
```

Comment:
I found that changing the sequence would cause circular dependency. Can you kindly double check by running one of TP unit test locally to see if it is still working? Thanks!

Reply:
good catch, yeah it indeed failed on CI before, I had to adjust the imports to avoid circular dependency, and now all should pass, i'll double verify on it https://github.com/pytorch/pytorch/pull/89969/files#diff-ee52b4a0e8716571b252d018ec260909345ecd9f636f7e56662866b386c81220L12